decidim-assemblies/app/packs/src/decidim/assemblies/admin/assemblies_list.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable require-jsdoc */
+
+class AdminAssembliesListComponent {
+  run() {
+    this.rebindArrows();
+  }
+
+  rebindArrows() {
+    this.unbindArrows();
+    this.bindArrows();
+  }
+
+  bindArrows() {
+    $("[data-arrow-up]").on("click", this._onClickUpArrow);
+    $("[data-arrow-down]").on("click", this._onClickDownArrow);
+  }
+
+  unbindArrows() {
+    $("[data-arrow-up]").off("click", this._onClickUpArrow);
+    $("[data-arrow-down]").off("click", this._onClickDownArrow);
+  }
+
+  _onClickDownArrow(event) {
+    event.preventDefault();
+
+    const $target = $(event.currentTarget);
+    const $assembly = $target.closest("[data-assembly-id]");
+
+    $target.toggleClass("hidden");
+    $assembly.find("[data-arrow-up]").toggleClass("hidden");
+  }
+
+  _onClickUpArrow(event) {
+    event.preventDefault();
+
+    const $target = $(event.currentTarget);
+    const $assembly = $target.closest("[data-assembly-id]");
+    const parentLevel = $assembly.data("level");
+
+    $target.toggleClass("hidden");
+    $assembly.find("[data-arrow-down]").toggleClass("hidden");
+
+    // iterate over all tr elements after the current tr element
+    $assembly.nextAll("tr").each((index, element) => {
+      const currentLevel = $(element).data("level");
+
+      if (currentLevel > parentLevel) {
```

Comment:
We may need to add here a breakpoint to avoid the behavior in the below movie.

In the movie, i am expanding "Nesciunt eum autem sint perferendis." assembly, to see the results, then i am expanding "new assembly", then having both expanded, i am collapsing the "New Assembly". As a result, all the expanded rows, getting removed. 

I would expect to be able to close only the "New Assembly" children.

[Screencast from 2024-12-05 11-59-35.webm](https://github.com/user-attachments/assets/a888dbfd-c54b-4957-86be-09a8e0ab6261)